### ISSUE TITLE
Geofence instances

### ISSUE THREAD

**Issue by virjay** [geoserver/geofence#65]
The password field under the 'instances' tab of Geofence is saved as clear text in PostgreSQL table. A md5 representation would help.

**Comment by etj** [comment]
Nope, the encoding should be reversible, since the original password should be sent to GeoServer.

**Comment by virjay** [comment]
OK. Use a technique similar to the PwEncoder.java. Passwords stored in clear text should be avoided.
